platform/o.n.bootstrap/src/org/netbeans/JarClassLoader.java

```diff
@@ -77,6 +78,22 @@
  */
 public class JarClassLoader extends ProxyClassLoader {
     private static Stamps cache;
+    private static final Name MULTI_RELEASE = new Name("Multi-Release");
+    private static final int BASE_VERSION = 8;
+    private static final int RUNTIME_VERSION;
+
+    static {
+        int version;
+        try {
+                Object runtimeVersion = Runtime.class.getMethod("version").invoke(null);
+                version = (int) runtimeVersion.getClass().getMethod("major").invoke(runtimeVersion);
+        }
+        catch (Throwable ex) {
+                version = BASE_VERSION;
+        }
+        RUNTIME_VERSION = version;
+    }    
```

Comment:
looks good to me but fix indentation please. It would be also good to use default NB formatting for brace positioning.

The rest of the file does *not* use this formatting

Reply:
Indentation fixed.